cfgov/tccp/jinja2/tccp/card.html

```diff
@@ -522,6 +522,123 @@ <h2>Cash advances</h2>
     {% else %}
         <p>This card does not offer cash advances.</p>
     {% endif %}
+    <h2>Fees</h2>
+    <dl class="m-list m-list__card-details">
+        {% if not card.periodic_fee_type %}
+            <dt>Annual fee</dt>
+            <dd>None</dd>
+        {% endif %}
+        {% if card.annual_fee %}
+            <dt>Annual fee</dt>
+            <dd>{{ '$' ~ card.annual_fee }}</dd>
+        {% endif %}
+        {% if card.monthly_fee %}
+            <dt>Monthly fee</dt>
+            <dd>{{ '$' ~ card.monthly_fee }}</dd>
+        {% endif %}
+        {% if card.weekly_fee %}
+            <dt>Weekly fee</dt>
+            <dd>{{ '$' ~ card.weekly_fee }}</dd>
+        {% endif %}
+        {% if card.other_periodic_fee_amount %}
+            <dt>{{ card.other_periodic_fee_name }}</dt>
+            <dd>{{ '$' ~ card.other_periodic_fee_amount ~ ', ' ~ card.other_periodic_fee_frequency | lower }}</dd>
+        {% endif %}
+        {% if card.fee_varies %}
+            <dt>Fee varies?</dt>
+            <dd>
+                {{ '$' ~ card.periodic_min ~ '–'  ~ card.periodic_max ~
+                    ' (' ~ card.fee_explanation | lower ~ ')'
```

Comment:
The use of `| lower` here could potentially cause problems if the explanation contains a word that should be capitalized. For example see [this card](http://localhost:8000/consumer-tools/credit-cards/explore-cards/cards/hawaiiusa-federal-credit-union-life-matters-cash-back-visa-credit-card/) - "signature visa product".